packages/remix-dev/cli/run.ts

```diff
@@ -303,6 +313,22 @@ export async function run(argv: string[] = process.argv.slice(2)) {
     case "setup":
       await commands.setup(input[1]);
       break;
+    case "codemod": {
+      if (!flags.dry) {
+        checkGitStatus(flags.force);
```

Comment:
I think it would be better to check the `git status` of the actual project directory where we are trying to run the transform. Right now, its checking the `git status` of the current working directory before even considering the project directory.

Reply:
That means we should move it to the `validateAnswers` function?